frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx

```diff
@@ -31,10 +31,10 @@ export function TrendTab({ view }: TrendTabProps): JSX.Element {
     const { groupsTaxonomicTypes } = useValues(groupsModel)
     const [isUsingFormulas, setIsUsingFormulas] = useState(filters.formula ? true : false)
     const lifecycles = [
-        { name: 'new', tooltip: 'Users that are new.' },
-        { name: 'resurrecting', tooltip: 'Users who were once active but became dormant, and are now active again.' },
-        { name: 'returning', tooltip: 'Users who consistently use the product.' },
-        { name: 'dormant', tooltip: 'Users who are inactive.' },
+        { name: 'new', tooltip: 'Users who signed up and did activity during period.' },
+        { name: 'returning', tooltip: 'Users who did activity both this and previous period.' },
+        { name: 'resurrecting', tooltip: 'Users who did activity this period but not the previous period.' },
```

Comment:
```suggestion
        { name: 'resurrecting', tooltip: 'Users who did the activity this period but did not do the activity on the previous period (i.e. were previously inactive for 1 or more periods).' },
```

Reply:
This is inaccurate - user is only counted as dormant if they did it previous period and not this. If they did action t-2 they are not counted as dormant.